File: api/src/main/java/com/delivery/api/config/health/HealthOpenApiController.java
package com.delivery.api.config.health;

import com.delivery.api.common.rabbitmq.Producer;
import lombok.RequiredArgsConstructor;
import lombok.extern.slf4j.Slf4j;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

@RestController
@Slf4j
@RequiredArgsConstructor
@RequestMapping("/open-api")
public class HealthOpenApiController {

    private final Producer producer;

    @GetMapping("/health")
    public void health() {
        log.info("health call");
    }

}

File: api/src/main/kotlin/com/delivery/api/config/health/HealthOpenApiController.kt
package com.delivery.api.config.health

import lombok.RequiredArgsConstructor
import org.slf4j.Logger
import org.slf4j.LoggerFactory
import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RequestMapping
import org.springframework.web.bind.annotation.RestController

@RestController
@RequiredArgsConstructor
@RequestMapping("/open-api")
class HealthOpenApiController() {

    private val logger: Logger = LoggerFactory.getLogger(this.javaClass)

    @GetMapping("/health")
    fun health() {
        logger.info("health call")
    }

}